packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -475,6 +536,28 @@ const Auth: React.FC<AuthProps> = ({
           </div>
         ) : null;
 
+      case "wallet":
+        return authConfig.walletEnabled && !otpId ? (
+          <div className={styles.passkeyContainer}>
```

Comment:
tiny nonblocking nit: we can probably rename `passkeyContainer` given it's been repurposed for multiple uses